docs/development/setup-vagrant.md

```diff
@@ -1091,6 +1092,38 @@ HOST_IP_ADDR 0.0.0.0
 (and restart the Vagrant guest), your host IP would be 0.0.0.0, a special value
 for the IP address that means any IP address can connect to your development server.
 
+### Customizing the virtual system settings
+
+When running Vagrant on top of Virtual Box or VM Ware Fusion, you can apply
+custom specifications for your VM.
+
+The default settings for the virtual machine's system is two cpus, with two GiB of memory.
```

Comment:
with two GiB of memory, which is sufficient to run everything in the development environment.

Also best to use the term "guest" rather than "virtual machine" because it might be a container, not a VM, with e.g. LXC